.github/CODEOWNERS

```diff
@@ -30,8 +30,8 @@
 
 # Generic fallback rules
 
-/addons/account*/ @odoo/rd-accounting
-/addons/l10n_*/ @odoo/rd-accounting
+# /addons/account*/ @odoo/rd-accounting
+# /addons/l10n_*/ @odoo/rd-accounting
```

Comment:
It's a bit strange to keep disabled rules, but I guess it does make the change and the review easier at every freeze so that's fair enough.